pandas/io/json/_json.py

```diff
@@ -1112,15 +1112,14 @@ def _parse_no_numpy(self):
             self.check_keys_split(decoded)
             self.obj = DataFrame(dtype=None, **decoded)
         elif orient == "index":
-            self.obj = (
-                DataFrame.from_dict(
-                    loads(json, precise_float=self.precise_float),
-                    dtype=None,
-                    orient="index",
-                )
-                .sort_index(axis="columns")
-                .sort_index(axis="index")
+            self.obj = DataFrame.from_dict(
+                loads(json, precise_float=self.precise_float),
+                dtype=None,
+                orient="index",
             )
+            if compat.PY35:
+                self.obj.sort_index(axis="columns", inplace=True)
```

Comment:
even though this is compatiblity code, don't use inplace

Reply:
Okay. Thank you for your good advice.